homestar-runtime/src/event_handler.rs

```diff
@@ -59,7 +59,9 @@ pub(crate) struct EventHandler<DB: Database> {
     query_senders: FnvHashMap<QueryId, (RequestResponseKey, P2PSender)>,
     connected_peers: FnvHashMap<PeerId, ConnectedPoint>,
     request_response_senders: FnvHashMap<RequestId, (RequestResponseKey, P2PSender)>,
+    rendezvous_cookies: FnvHashMap<PeerId, Cookie>,
     ws_msg_sender: ws::Notifier,
+    external_address_limit: usize,
```

Comment:
this setting should probably just be u32 or even u16 (always positive?). We can always use usize at the call site.